src/main/java/org/testng/JarFileUtils.java

```diff
@@ -27,13 +27,20 @@
     private final String xmlPathInJar;
     private final List<String> testNames;
     private final List<XmlSuite> suites = Lists.newLinkedList();
+    private final XmlSuite.ParallelMode mode;
 
     JarFileUtils(IPostProcessor processor, String xmlPathInJar, List<String> testNames) {
+        this(processor, xmlPathInJar, testNames, null);
```

Comment:
Maybe `ParallelMode.NONE` instead of null as default value

Reply:
Fixed